js/src/api/format/input.js

```diff
@@ -127,6 +127,18 @@ export function inNumber16 (number) {
   return inHex(bn.toString(16));
 }
 
+export function inOptionsCondition (condition) {
```

Comment:
Honestly don't have the space to add it here for release. Hence the comment in the PR - I need tests for all of these.